app/src/main/java/io/zenandroid/onlinego/ui/screens/main/MainActivity.kt

```diff
@@ -221,6 +223,12 @@ class MainActivity : AppCompatActivity(), MainContract.View {
         super.onResume()
     }
 
+    override fun onConfigurationChanged(newConfig: Configuration) {
```

Comment:
Oh, we're getting fancy now :)

Reply:
Hmmm... I think this is causing issues sometimes. If I swap between light and dark a few times I end up in some weird states at some point, for example with the dark text on dark background (!) or dark board texture on light theme. I think we should let Android restart the app on the color change as it used to do before we added the configuration="uiMode" in the manifest. See screenshots below:

![Screenshot_1590582256](https://user-images.githubusercontent.com/2835035/83018535-6491e480-a01d-11ea-91f6-89b779571d75.png)
![Screenshot_1590582477](https://user-images.githubusercontent.com/2835035/83018862-e255f000-a01d-11ea-83c9-8035a3c71e33.png)

Reply:
resolved in the last commit; kept the ==null , removed onConfigurationChanged and moved the texture load to the activity onCreate (only called at startup and on config changes)